Lib9c/Model/Config/ArenaConfig.cs

```diff
@@ -0,0 +1,61 @@
+using Nekoyume.TableData;
+
+namespace Nekoyume.Model.Config
+{
+    public class ArenaConfig : IConfig
+    {
+        public int ArenaScoreDefault { get; private set; }
+        public int ArenaChallengeCountMax { get; private set; } 
+        public int DailyArenaInterval { get; private set; }
+        public int WeeklyArenaInterval { get; private set; }
+
+        public ArenaConfig()
+        {
+        }
+
+        public ArenaConfig(string csv)
+        {
+            var sheet = new ArenaConfigSheet();
+            sheet.Set(csv);
+            foreach (var row in sheet.Values)
+            {
+                Update(row);
+            }
+        }
+
+        public ArenaConfig(ArenaConfigSheet sheet)
+        {
+            foreach (var row in sheet.Values)
+            {
+                Update(row);
+            }
+        }
+
+        public void Set(ArenaConfigSheet sheet)
+        {
+            foreach (var row in sheet)
```

Comment:
Is this okay to be used without explicit ordering?

Reply:
It seems like you can use `sheet.OrderedList`.

Reply:
Order of the elements is not important in this situation.